app/bundles/CoreBundle/Controller/AjaxController.php

```diff
@@ -301,6 +301,8 @@ protected function togglePublishStatusAction(Request $request)
             }
         }
 
+        $dataArray['flashes'] = $this->getFlashContent();
```

Comment:
I'm not able to cover this in reasonable time. This and other cases.
We are not able to cover Controllers in general because of complexity.
What we could do with this? @escopecz

Reply:
It can be tested with a functional test if we decide this line must be tested.